### DISCUSSION TITLE
Lumi and xAPI

### DISCUSSION THREAD

**Discussion by rrredirect** [Lumieducation/Lumi#2253] [Q&A]
Hello,

I just discovered Lumi and I am very excited by what it has to offer. My company wishes to send the xAPI statements generated by H5P interactions to our LRS. We've considered self-hosting H5P, but Lumi looks even better. Does Lumi allow for H5P xAPI statements to be generated? We would like to embed the content within a Rise or Storyline project. I do know some code, so I do not mind a workaround that requires some code. 
Thank you,
-Josh

**Comment by s9332084** [comment]
did you get already answer to this issue?

**Comment by rrredirect** [comment]
I did not receive an answer yet
